packages/validator/docs/actions.test.ts

```diff
@@ -0,0 +1,234 @@
+import * as convertor from '../src/convertor';
+
+describe('actions (docs)', () => {
+  it('copy action', () => {
+    const template = {
+      company: {},
+      inventory: {
+        __arrayItem: {
+          name: {},
+          quantity: {},
+        },
+        __actions: [
+          {
+            __copy: {
+              __target: "[ 'backups', '[[ \\'/\\', \\'company\\' ]]' ]",
```

Comment:
> __target: `[ "backups", "[[ '/', 'company' ]]" ]`,

is not valid json

> Why can't I just write [["/company"]]?

some keys in specifications might be endpoints like this one: `/endpt/path`, which would break it